### ISSUE TITLE
`README.md` ?

### ISSUE THREAD

**Issue by nelsonic** [MikeRund/OpenAir#1]
At present this repo is a bit bare ... 🤷‍♂️ 
<img width="1174" alt="image" src="https://github.com/user-attachments/assets/092857c2-430f-4657-95db-183f8b89eb9e">

Could be worth adding a `README.md` with a few screenshots, badges and  explanation. 💭 
Especially for the people who have `JAVA` PTSD and won't run the code on their `localhost` to _see_ it working. 😜 
Especially cool would be a `Video` or `GIF` of the app. 📺

**Comment by MikeRund** [comment]
Yes! Very good point, I need to extend my flashy new readme skills to my projects. Fortunately I do have a video and screenshots kicking around so won't be needing to dig up the old android

**Comment by nelsonic** [comment]
Sweet! 🍭 
It will be a _muuuuch_ better than a bare repo at showcasing all the effort you put in! 😍 
Then you can _link_ directly to it from your `CV` 🧾 
<img width="840" alt="image" src="https://github.com/user-attachments/assets/0562e7c5-2bc9-4a63-a359-cdd2f1fb3d4a">
And people _reading_ your `CV` can get a _waaaaay_ bettererer idea of your skills. 🧙 🪄 ✨
